### PR TITLE
Bump to v2.8.2

### PR DESCRIPTION
**Pull request by alicethorne-ab**
Bump to v2.8.2.

`mpdev verify` succeeded:

```
=== VERIFICATION STATUS ===
PASSED
===========================
```

### CODE REVIEW

File: op-scim-bridge/Makefile [hunk 1]
```diff
@@ -48,5 +48,8 @@ app/build:: .build/check/docker .build/op-scim/deployer .build/op-scim/deps
 .build/op-scim/deps:
 	./push-dep-image-tags.sh
 
+.build/op-scim:
```
Comment (alicethorne-ab) [inline op-scim-bridge/Makefile:51]: Added this build step as the directory not existing was causing errors on my freshly-checked-out repo.

**Review by chasdl — DISMISSED** [review]
Looks good!

**Comment by alicethorne-ab** [comment]
Submitted to GCP. Holding off on merge until the deployment is fully published.

File: op-scim-bridge/chart/op-scim-bridge-mp/Chart.lock [hunk 2]
```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: op-scim-bridge
   repository: https://1password.github.io/op-scim-helm/
-  version: 2.10.1
+  version: 2.10.3
 - name: common
   repository: https://charts.bitnami.com/bitnami
   version: 1.17.1
-digest: sha256:dd5c3f18d7a7b070e36555c754da14064afd7bf2f69d66f9265ded11531907ce
-generated: "2023-05-09T08:48:21.931439-04:00"
```
Comment (grellyd) [inline op-scim-bridge/chart/op-scim-bridge-mp/Chart.lock:9]: Does this generated value need to be updated in this MR?

File: op-scim-bridge/chart/op-scim-bridge-mp/values.yaml [hunk 3]
```diff
@@ -9,3 +9,9 @@
 # otherwise the chart name "op-scim-bridge-mp" is used instead and the end result is "Referenced object not found"
 # is shown in the GCP console instead of the external IP of the service
 nameOverride: op-scim-bridge
+scim:
+  credentials:
+    volume:
+      enabled: true
+    secrets:
+      enabled: false
```
Comment (grellyd) [inline op-scim-bridge/chart/op-scim-bridge-mp/values.yaml:17]: Perhaps we should explicitly set the storage class here [like I have done in the DO version](https://github.com/digitalocean/marketplace-kubernetes/pull/469/files#diff-756df5eccfd3b933576044fd4dff3a736cba201cc9369d5772c6a0b071f75fbdR5). I believe the correct value for GCP is `"-"`

**Review by grellyd — APPROVED** [review]
Worked on this with Alice. Looks good with a few comments. Thanks for the changes.

File: op-scim-bridge/chart/op-scim-bridge-mp/Chart.lock (hunk 2 above)
Comment (alicethorne-ab) [reply op-scim-bridge/chart/op-scim-bridge-mp/Chart.lock:9]: Yes, it references the upstream chart from `op-scim-helm` and needs to be updated with each version bump.

**Comment by chasdl** [comment]
Closing with https://github.com/1Password/op-scim-gcp-marketplace/pull/77

**Pull request closed without merging**